pages/common/errno.md

```diff
@@ -0,0 +1,20 @@
+# errno
+
+> Look up errno names and descriptions.
+> More information: <https://joeyh.name/code/moreutils/>.
+
+- Lookup errno description by name or code:
+
+`errno {{name_or_code}}`
```

Comment:
```suggestion
`errno {{name|code}}`
```

It's better to use pipe delimiter instead of `or`. The problem arises when we have to separate multi word values.